api/config/config_perf.json

```diff
@@ -135,5 +135,9 @@
   "reporting": {
     "dataset": "reporting_perf",
     "maxRowsPerInsert": 2500
+  },
+  "ras": {
+    "host": "not implemented",
```

Comment:
nit: personally I'd just prefer empty string here, I'd say that's the usual convention we use for new flags which don't have values yet in higher environments